studybuddies/src/app/components/login/login.component.ts

```diff
@@ -1,4 +1,9 @@
 import { Component, OnInit } from '@angular/core';
+import { User } from "../../models/user";
+import { LoginService } from "../../services/login/login.service";
+import { Router } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7199829)
```suggestion
import { Router } from "@angular/router";
```